### PR TITLE
Propose fixes to issues #125,#105

### PR DESCRIPTION
**Pull request by lana-wu**
Replace Packages "revision" with "version" and remove leaf "previous-version""nbc-changes"

### CODE REVIEW

File: yang-packages/draft-ietf-netmod-yang-packages.txt [hunk 1]
```diff
@@ -516,15 +509,11 @@ Internet-Draft                YANG Packages                February 2022
    the package (included packages or modules) can be used to help
    determine the scope of changes being made.
 
-5.2.3.  Revision history
+5.2.3.  Version history
```
Comment (jsterne) [inline yang-packages/draft-ietf-netmod-yang-packages.txt:512]: Remove this section completely.

File: yang-packages/draft-ietf-netmod-yang-packages.txt [hunk 2]
```diff
@@ -1159,7 +1146,7 @@ Internet-Draft                YANG Packages                February 2022
      // and remove this note.
      // RFC Ed.: replace XXXX with actual RFC number and remove this
      // note.
-     revision 2021-10-25 {
+     revision 2022-02-08 {
        rev:revision-label 0.2.0;
```
Comment (Reshad-Rahman) [inline yang-packages/draft-ietf-netmod-yang-packages.txt:1150]: Unrelated to this change: the revision-label is supposed to have the draft ID.

Comment (jsterne) [reply yang-packages/draft-ietf-netmod-yang-packages.txt:1150]: I had fixed this to 0.3.0-draft-ietf-netmod-yang-packages-03; in my previous PR. Maybe just use the same for all 4 other modules.

File: yang-packages/draft-ietf-netmod-yang-packages.xml [hunk 3]
```diff
@@ -1137,23 +1124,6 @@ module ietf-yang-package-types {
...
-      default false;
-      description
-        "Indicates whether the defined package version contains
-         non-backwards-compatible changes relative to the package
-         version defined in the 'previous-version' leaf.";
-    }
-
     leaf-list tag {
       type tags:tag;
       description
```
Comment (jsterne) [inline yang-packages/draft-ietf-netmod-yang-packages.xml:1129]: In the "list included-package {" below, change this sentence:

         If included packages implement different revisions or
         versions of the same module, then an explicit entry in the

to this (drop the "or versions"):

         If included packages implement different revisions 
         of the same module, then an explicit entry in the

Comment (jsterne) [inline yang-packages/draft-ietf-netmod-yang-packages.xml:1129]: Below, in  "leaf-list replaces-version {", change "revision" to "version" in this sentence:
        description
          "Gives the version of an included package version that
           is replaced by this included package revision.";
      }

Below, remove leaf nbc-modified

**Pull request merged**